src/transaction/constants.ts

```diff
@@ -0,0 +1,3 @@
+export const SEND_CALLS_NOT_SUPPORTED_ERROR = 'SEND_CALLS_NOT_SUPPORTED_ERROR';
+export const METHOD_NOT_SUPPORTED_ERROR_SUBSTRING =
+  'this request method is not supported'; // most likely EOA
```

Comment:
If this message is displayed, we probably want the `t` upper case. as `This request`